tests/files/test-inventory

```diff
@@ -1,4 +1,2 @@
-test-bastion ansible_ssh_host=127.0.0.1 ansible_ssh_user=ubuntu
```

Comment:
This inventory isnt really used for anything other than syntax checking our playbooks,  so we dont actually need the connection info